pkg/kubelet/kubelet.go

```diff
@@ -319,6 +327,26 @@ type Dependencies struct {
 	useLegacyCadvisorStats bool
 }
 
+// newCrashLoopBackOff configures the backoff maximum to be used
+// by kubelet for container restarts depending on the alpha gates
+// and kubelet configuration set
+func newCrashLoopBackOff(kubeCfg *kubeletconfiginternal.KubeletConfiguration) (time.Duration, time.Duration) {
+	boMax := MaxCrashLoopBackOff
+	boInitial := containerBackOffPeriod
```

Comment:
nit: can you rename this constant too, for consistency (unless it's used elsewhere?)
```suggestion
	boInitial := initialCrashLoopBackOff
```

Reply:
you're right, and I moved it up with the other crashloop related rconstants like you had asked in a prior comment thread